### PR TITLE
Improve API docs and add to separate folder

### PR DESCRIPTION
**Pull request by JakeSCahill**
Added which parameters are required in the `api.yaml` file.

Added an `env.json` file to use to autogenerate the markdown content.

Added the autogenerated markdown file, which will be hosted on the documentation portal.

### CODE REVIEW

File: plugins/webapi/docs/api-reference.md [hunk 1]
```diff
@@ -0,0 +1,1053 @@
...
+  "address": "string",
+  "data": "string"
+}
+```
+
+<h3 id="post__broadcastdata-parameters">Body parameters</h3>
+
+|**Name**|**Type**|**Required**|**Description**|
+|---|---|---|---|---|
+|body|object|true|Request object|
```
Comment (luca-moser) [inline plugins/webapi/docs/api-reference.md]: this looks weird in rendered form, is this wanted?

Comment (JakeSCahill) [reply plugins/webapi/docs/api-reference.md]: Looks like there's one too many columns

Comment (JakeSCahill) [reply plugins/webapi/docs/api-reference.md]: Fixed them 👍

Comment (luca-moser) [reply plugins/webapi/docs/api-reference.md]: cool thanks

**Pull request merged**